OlinkAnalyze/inst/WORDLIST

```diff
@@ -0,0 +1 @@
+Benjamini
```

Comment:
There will likely be others to add too but definitely a good start.

Reply:
I just added this one as a proof of concept that the WORDLIST is used in spellcheck...